### ISSUE TITLE
Load workflow fail after update frontend 1.2.40

### ISSUE THREAD

**Issue by K-O-N-B** [comfyanonymous/ComfyUI#4743]
### Your question

can anybody tell me where is the problem? or which node make this error?
![image](https://github.com/user-attachments/assets/893de6a5-1d86-483e-9855-965743dad3e1)
![image](https://github.com/user-attachments/assets/93dce3a5-5646-42e1-98f2-df4dd18c188a)
![image](https://github.com/user-attachments/assets/a0b67306-c088-4343-9123-efe83c28992f)

### Logs

```powershell
[]
[]
data_path:  H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-mixlab-nodes\data
#read_workflow_json_files_all H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-mixlab-nodes\app\
[START] Security scan
[DONE] Security scan
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-09-02 11:06:14.821765
** Platform: Windows
** Python version: 3.11.6 | packaged by conda-forge | (main, Oct  3 2023, 10:29:11) [MSC v.1935 64 bit (AMD64)]
** Python executable: H:\ComfyUI-qiuye\ComfyUI\.ext\python.exe
** ComfyUI Path: H:\ComfyUI-qiuye\ComfyUI
** Log path: H:\ComfyUI-qiuye\ComfyUI\comfyui.log

Prestartup times for custom nodes:
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\rgthree-comfy
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Easy-Use
   6.6 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 24564 MB, total RAM 32525 MB
pytorch version: 2.2.0+cu121
xformers version: 0.0.24
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4090 : cudaMallocAsync
Using xformers cross attention
[Prompt Server] web root: H:\ComfyUI-qiuye\ComfyUI\web
Adding extra search path checkpoints H://SD-webui-aki-qiuye\models/Stable-diffusion
Adding extra search path configs H://SD-webui-aki-qiuye\models/Stable-diffusion
Adding extra search path vae H://SD-webui-aki-qiuye\models/VAE
Adding extra search path loras H://SD-webui-aki-qiuye\models/Lora
Adding extra search path loras H://SD-webui-aki-qiuye\models/LyCORIS
Adding extra search path upscale_models H://SD-webui-aki-qiuye\models/ESRGAN
Adding extra search path upscale_models H://SD-webui-aki-qiuye\models/RealESRGAN
Adding extra search path upscale_models H://SD-webui-aki-qiuye\models/SwinIR
Adding extra search path embeddings H://SD-webui-aki-qiuye\embeddings
Adding extra search path hypernetworks H://SD-webui-aki-qiuye\models/hypernetworks
Adding extra search path controlnet H://SD-webui-aki-qiuye/extensions/sd-webui-controlnet/models
[Allor]: 0 nodes were overridden.
[Allor]: 12 modules were enabled.
[Allor]: 98 nodes were loaded.
Adding H:\ComfyUI-qiuye\ComfyUI\custom_nodes to sys.path
Efficiency Nodes: Attempting to add Control Net options to the 'HiRes-Fix Script' Node (comfyui_controlnet_aux add-on)...Success!
Loaded Efficiency nodes from H:\ComfyUI-qiuye\ComfyUI\custom_nodes\efficiency-nodes-comfyui
Loaded ControlNetPreprocessors nodes from H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui_controlnet_aux
Loaded AdvancedControlNet nodes from H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet
Could not find AnimateDiff nodes
Loaded IPAdapter nodes from H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_IPAdapter_plus
Loaded VideoHelperSuite from H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite
### Loading: ComfyUI-Impact-Pack (V7.4.1)
[WARN] ComfyUI-Impact-Pack: custom_wildcards path not found: H:\ComfyUI_qiuye\custom_nodes\ComfyUI-Impact-Pack\custom_wildcards. Using default path.
### Loading: ComfyUI-Impact-Pack (Subpack: V0.6)
### Loading: ComfyUI-Impact-Pack (V7.4.1)
Loaded ImpactPack nodes from H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Impact-Pack[Impact Pack] Wildcards loading done.

[Impact Pack] Wildcards loading done.
[Crystools INFO] Crystools version: 1.12.0
[Crystools INFO] CPU: 12th Gen Intel(R) Core(TM) i5-12400 - Arch: AMD64 - OS: Windows 10
[Crystools INFO] GPU/s:
[Crystools INFO] 0) NVIDIA GeForce RTX 4090
[Crystools INFO] NVIDIA Driver: 560.94
[ComfyUI-Easy-Use] server: v1.2.2 Loaded
[ComfyUI-Easy-Use] web root: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Easy-Use\web_version/v2 Loaded
Total VRAM 24564 MB, total RAM 32525 MB
pytorch version: 2.2.0+cu121
xformers version: 0.0.24
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4090 : cudaMallocAsync
### Loading: ComfyUI-Impact-Pack (V7.4.1)
[Impact Pack] Wildcards loading done.
### Loading: ComfyUI-Inspire-Pack (V1.1)
A new version of Albumentations is available: 1.4.14 (you have 1.4.11). Upgrade using: pip install --upgrade albumentations
theUpsiders Logic Nodes: Loaded
### Loading: ComfyUI-Manager (V2.50.3)
### ComfyUI Revision: 2635 [c90459eb] | Released on '2024-08-30'
--------------
 ### Mixlab Nodes: Loaded
ChatGPT.available True
edit_mask.available True
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
LaMaInpainting.available True
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
ClipInterrogator.available True
PromptGenerate.available True
ChinesePrompt.available True
RembgNode_.available True
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
TripoSR.available
 --------------
ComfyUI-N-Sidebar is loading...
No 'sb_wf_path' in settings.json. Using default path.
Use STYLE(weight_interpretation, normalization) at the start of a prompt to use advanced encodings
Weight interpretations available: comfy,A1111,compel,comfy++,down_weight,perp
Normalization types available: none,mean,length,length+mean
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
[SD Prompt Reader] Node version: 1.3.3
[SD Prompt Reader] Core version: 1.3.4.post1
Failed to auto update `Quality of Life Suit`
QualityOfLifeSuit_Omar92_DIR: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-QualityOfLifeSuit_Omar92
[ReActor] - STATUS - Running v0.5.1-a6 in ComfyUI
Torch version: 2.2.0+cu121
Imported node: DT_Flatten_Colors
Imported node: DT_FontText
Imported node: DT_GenerateNoise
Imported node: DT_Glitch_This
Imported node: DT_Hue_Rotation
Imported node: DT_Load_Picture_Index
Imported node: DT_PILGram
Imported node: DT_Pixel_Sort
Imported node: DT_Play_Sound_At_Execution
Imported node: DT_PromptGen
Imported node: DT_Solid_Color
Imported node: DT_Swap_Color_Mode
Imported node: DT_Flatten_Colors
Imported node: DT_FontText
Imported node: DT_GenerateNoise
Imported node: DT_Glitch_This
Imported node: DT_Hue_Rotation
Imported node: DT_Load_Picture_Index
Imported node: DT_PILGram
Imported node: DT_Pixel_Sort
Imported node: DT_Play_Sound_At_Execution
Imported node: DT_PromptGen
Imported node: DT_Solid_Color
Imported node: DT_Swap_Color_Mode
(pysssss:WD14Tagger) [DEBUG] Available ORT providers: CPUExecutionProvider
(pysssss:WD14Tagger) [DEBUG] Using ORT providers: CUDAExecutionProvider, CPUExecutionProvider
Workspace manager - Openning file hash dict
🦄🦄Loading: Workspace Manager (V2.1.0)
[09/02/24 11:06:45] WARNING  Your inference package version      __init__.py:35
                             0.9.17 is out of date! Please
                             upgrade to version 0.17.0 of
                             inference for the latest features
                             and bug fixes by running `pip
                             install --upgrade inference`.
SupervisionWarnings: BoxAnnotator is deprecated: `BoxAnnotator` is deprecated and will be removed in `supervision-0.22.0`. Use `BoundingBoxAnnotator` and `LabelAnnotator` instead
3D OpenPose Editor: Loaded
------------------------------------------
Comfyroll Studio v1.76 :  175 Nodes Loaded
------------------------------------------
** For changes, please see patch notes at https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes/blob/main/Patch_Notes.md
** For help, please see the wiki at https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes/wiki
------------------------------------------
### [START] ComfyUI AlekPet Nodes v1.0.23 ###
Node -> ArgosTranslateNode: ArgosTranslateCLIPTextEncodeNode, ArgosTranslateTextNode [Loading]
Node -> DeepTranslatorNode: DeepTranslatorCLIPTextEncodeNode, DeepTranslatorTextNode [Loading]
Node -> GoogleTranslateNode: GoogleTranslateCLIPTextEncodeNode, GoogleTranslateTextNode [Loading]
Node -> ExtrasNode: PreviewTextNode, HexToHueNode, ColorsCorrectNode [Loading]
Node -> PoseNode: PoseNode [Loading]
Node -> IDENode: IDENode [Loading]
Node -> PainterNode: PainterNode [Loading]
### [END] ComfyUI AlekPet Nodes ###
Davemane42 Custom Nodes: Loaded
FizzleDorf Custom Nodes: Loaded
Ib Custom Nodes: Loaded
# 😺dzNodes: LayerStyle -> Cannot import name 'guidedFilter' from 'cv2.ximgproc'
A few nodes cannot works properly, while most nodes are not affected. Please REINSTALL package 'opencv-contrib-python'.
For detail refer to https://github.com/chflame163/ComfyUI_LayerStyle/issues/5
# 😺dzNodes: LayerStyle -> Warning: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_LayerStyle\resource_dir.ini not found, default directory to be used.
# 😺dzNodes: LayerStyle -> Find 1 LUTs in H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_LayerStyle\lut
# 😺dzNodes: LayerStyle -> Find 3 Fonts in H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_LayerStyle\font
Traceback (most recent call last):
  File "H:\ComfyUI-qiuye\ComfyUI\nodes.py", line 1993, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Native_DynamiCrafter\__init__.py", line 1, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Native_DynamiCrafter\nodes.py", line 11, in <module>
    from .utils.model_utils import DynamiCrafterBase, DYNAMICRAFTER_CONFIG, \
  File "H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Native_DynamiCrafter\utils\model_utils.py", line 9, in <module>
    from comfy import sd2_clip
ImportError: cannot import name 'sd2_clip' from 'comfy' (unknown location)

Cannot import H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Native_DynamiCrafter module for custom nodes: cannot import name 'sd2_clip' from 'comfy' (unknown location)
[tinyterraNodes] Loaded
[comfy_mtb] | INFO -> loaded 85 nodes successfuly
FaceDetailer: Model directory already exists
FaceDetailer: Model already exists
WARNING: Could not find OPENAI_API_KEY in .env, disabling gpt prompt generation.
Nvidia APEX normalization not installed, using PyTorch LayerNorm

[rgthree] Loaded 41 exciting nodes.
[rgthree] NOTE: Will NOT use rgthree's optimized recursive execution as ComfyUI has changed.

WAS Node Suite: BlenderNeko's Advanced CLIP Text Encode found, attempting to enable `CLIPTextEncode` support.
WAS Node Suite: `CLIPTextEncode (BlenderNeko Advanced + NSP)` node enabled under `WAS Suite/Conditioning` menu.
WAS Node Suite: OpenCV Python FFMPEG support is enabled
WAS Node Suite Warning: `ffmpeg_bin_path` is not set in `H:\ComfyUI-qiuye\ComfyUI\custom_nodes\was-node-suite-comfyui\was_suite_config.json` config file. Will attempt to use system ffmpeg binaries if available.
WAS Node Suite: Finished. Loaded 217 nodes successfully.

        "Art is the only way to run away without leaving home." - Twyla Tharp


Import times for custom nodes:
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\SDXLAspectRatio.py
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\Nk_vaeswitch.py
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\Nk_imgInputSwitch3Way.py
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Photopea
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\Nk_4way Latent Switch.py
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\TTP_tile_preprocessor_v5.py
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\NK_4way Image Switch_v2.py
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\websocket_image_save.py
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_IPAdapter_plus
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\efficiency-nodes-comfyui
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_3dPoseEditor
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-sampler-lcm-alternative
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\style_aligned_comfy
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-deepcache
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-popup_preview
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Logic
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\sdxl_prompt_styler
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_GradientDeepShrink
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Image-Selector
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\cg-use-everywhere
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Noise
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_ADV_CLIP_emb
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\Comfyui_MiniCPMv2_6-prompt-generator
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Ib_CustomNodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\lora-info
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Cutoff
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-prompt-composer
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-BRIA_AI-RMBG
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_TiledKSampler
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_cspnodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Inpaint-CropAndStitch
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-N-Sidebar
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\cg-image-picker
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUi_NNLatentUpscale
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Lora-Auto-Trigger-Words
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\stability-ComfyUI-nodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\masquerade-nodes-comfyui
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Dave_CustomNode
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfy_sd_krita_plugin_nodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui_controlnet_aux
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\IPAdapter-ComfyUI
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-WD14-Tagger
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-IC-Light
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_MiniCPM-V-2_6-int4
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_InstantID
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\Comfyui-StableSR
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_VisualStylePrompting
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-AutomaticCFG
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyI2I
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\Comfyui_CXH_joy_caption
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-inpaint-nodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-QualityOfLifeSuit_Omar92
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_FaceShaper
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-tooling-nodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Impact-Pack
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-TiledDiffusion
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Video-Matting
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-layerdiffusion
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Kolors-MZ
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-various
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_essentials
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyMath
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Custom-Scripts
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\mikey_nodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\DZ-FaceDetailer
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-DynamiCrafterWrapper
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_FizzNodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Flowty-LDSR
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\steerable-motion
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\Derfuu_ComfyUI_ModdedNodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Frame-Interpolation
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\x-flux-comfyui
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-segment-anything-2
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-KJNodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Florence2
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-GGUF
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfy-image-saver
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\rgthree-comfy
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\OneButtonPrompt
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-AnimateDiff-Evolved
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-SUPIR
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_smZNodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-EasyAnimateWrapper
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Comfyroll_CustomNodes
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-prompt-reader-node
   0.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\facerestore_cf
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfy_mtb
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-workspace-manager
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Inspire-Pack
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-prompt-control
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_FaceAnalysis
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_tinyterraNodes
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\sd-ppp
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\eden_comfy_pipelines
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-BrushNet-Wrapper
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui_segment_anything
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\PuLID_ComfyUI
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-IF_AI_tools
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_LayerStyle
   0.1 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\CharacterFaceSwap
   0.2 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-reactor-node
   0.2 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Gemini
   0.2 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Vextra-Nodes
   0.3 seconds (IMPORT FAILED): H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Native_DynamiCrafter
   0.4 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-ollama
   0.5 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-DynamiCrafter
   0.7 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Easy-Use
   0.7 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Manager
   0.7 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-APISR
   0.8 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Allor
   1.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-mixlab-nodes
   1.3 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM
   1.4 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Inspyrenet-Rembg
   1.7 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-art-venture
   4.4 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI_Custom_Nodes_AlekPet
   6.7 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\was-node-suite-comfyui
   7.0 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Crystools
   9.2 seconds: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-BrushNet

https_key OK:  H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-mixlab-nodes\https\certificate.crt H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-mixlab-nodes\https\private.key




Starting server
To see the GUI go to: http://127.0.0.1:8188
To see the GUI go to: https://127.0.0.1:8189
FETCH DATA from: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
#clip_model_path chatml None
#model
start_local_llm error
HTTP Request: GET https://api.openai.com/v1/models "HTTP/1.1 401 Unauthorized"
Error: OpenAI API key is invalid OpenAI features wont work for you
QualityOfLifeSuit_Omar92::NSP ready
[comfy_mtb] | INFO -> Found multiple match, we will pick the last H://SD-webui-aki-qiuye\models/SwinIR
['H:\\ComfyUI-qiuye\\ComfyUI\\models\\upscale_models', 'H://SD-webui-aki-qiuye\\models/ESRGAN', 'H://SD-webui-aki-qiuye\\models/RealESRGAN', 'H://SD-webui-aki-qiuye\\models/SwinIR']
[]
[]
[]
[]
HTTP Request: GET http://127.0.0.1:11434/api/tags "HTTP/1.1 200 OK"
#read_workflow_json_files_all H:\ComfyUI-qiuye\ComfyUI\custom_nodes\comfyui-mixlab-nodes\app\
FETCH DATA from: H:\ComfyUI-qiuye\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
[]
[]
#clip_model_path chatml None
#model
start_local_llm error
[]
[]
```

**Comment by allmelgr** [comment]
the same problem

**Comment by allmelgr** [comment]
@K-O-N-B   used a previous version of ComfyUI and saved the workflow, which made it effective. However, I think some features are not compatible with the new version.

**Comment by iamrohitanshu** [comment]
Are other workflows working fine? Like the default workflow or any other simple workflow?
If yes, then that might be a problem with the custom nodes, and you'd wanna raise an issue there.

**Comment by K-O-N-B** [comment]
> Are other workflows working fine? Like the default workflow or any other simple workflow? If yes, then that might be a problem with the custom nodes, and you'd wanna raise an issue there.

some simple workflow work fine, but I don't know coding......So, don't know which custom nodes make this error....  *——*

**Comment by Dam88en** [comment]
i feel your pain, this happened to me too. how can we fix this?

**Comment by joeluo** [comment]
I found 'Use Everywhere(UE Nodes)' will cause issue. Uninstall or update this custom node may be solve the issue.

**Comment by huchenlei** [comment]
Update your ComfyUI-Custom-Scripts extension to include following commit.
https://github.com/pythongosssss/ComfyUI-Custom-Scripts/commit/013894fa8278068d900ef56794ea66fd5bba33c6

**Comment by huchenlei** [comment]
This is not a core issue.

**Comment by huchenlei** [comment]
Issue transferred to https://github.com/Comfy-Org/ComfyUI_frontend/issues/710. Please follow the update there. Thanks!

**Comment by allmelgr** [comment]
@K-O-N-B 
maybe cause by es6 update? fix after updating custom node ComfyUI-Custom-Scripts
https://github.com/pythongosssss/ComfyUI-Custom-Scripts/commit/013894fa8278068d900ef56794ea66fd5bba33c6

**Comment by K-O-N-B** [comment]
> @K-O-N-B maybe cause by es6 update? fix after updating custom node ComfyUI-Custom-Scripts [pythongosssss/ComfyUI-Custom-Scripts@013894f](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/commit/013894fa8278068d900ef56794ea66fd5bba33c6)

this error still here, drive me carzy

Custom-scripts last update at june, has it been updated recently?

**Comment by huchenlei** [comment]
> > @K-O-N-B maybe cause by es6 update? fix after updating custom node ComfyUI-Custom-Scripts [pythongosssss/ComfyUI-Custom-Scripts@013894f](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/commit/013894fa8278068d900ef56794ea66fd5bba33c6)
> 
> this error still here, drive me carzy
> 
> Custom-scripts last update at june, has it been updated recently?
> 
> The workflow seems ok after disable Custom-scripts node

The error is caused by core. Not custom node. The fix is available in nightly release 1.2.44. Follow https://github.com/Comfy-Org/ComfyUI_frontend#nightly-release to opt-in nightly release.

**Comment by DmitryFX** [comment]
Kind of similar problem: Error loading workflow file 'Flux Q8.json': 404 Not Found
Meanwhile the file exists on the disk.

![image](https://github.com/user-attachments/assets/5571eaf6-54f7-4c56-9a6e-f7f9add2ff57)

`Request URL:
http://127.0.0.1:8188/api/userdata/workflows%2FFlux%20Q8.json
Request Method:
GET
Status Code:
404 Not Found`

`api.ts:73 
 GET http://127.0.0.1:8188/api/userdata/workflows%2F.index.json net::ERR_ABORTED 404 (Not Found)

api.ts:73 
 GET http://127.0.0.1:8188/api/userdata/workflows%2FFlux%20Q8.json 404 (Not Found)`


ComfyUI: 2645[f067ad](2024-09-03) (**by the way, how this version number is connected to the release version here on git?**)
Manager: V2.50.3
